### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by JasemDuncan**
In this milestone, all the styling where add, after making a functional test.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @JasemDuncan 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# WEBPACK BOILERPLATE
+# Leaderboard
 
-> This is a template to start a JS project using webpack.
+> This is an app where you can check and submits scores usings APIs.
```
Comment (mikenath223) [inline README.md:5]: - [x] Kindly deploy the project online and include the live link in the readme.

Comment (JasemDuncan) [reply README.md:5]: Thanks, Sir. Done.

**Review by rotimiazeez — APPROVED** [review]
# Hi @JasemDuncan , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**